life_expectancy/tests/test_cleaning.py

```diff
@@ -1,20 +1,51 @@
 """Tests for the cleaning module"""
 import pandas as pd
+import pytest
+from life_expectancy.cleaning import clean_data, save_data, load_data
 
 
-from life_expectancy.cleaning import main
-from . import OUTPUT_DIR
+def test_clean_data(
+    pt_life_expectancy_input : pd.DataFrame,
+    pt_life_expectancy_expected : pd.DataFrame) -> None :
+    """Use clean_data function.
+    Assert top 5 lines of created and expected dataframes."""
 
+    pt_life_expectancy_input = clean_data(pt_life_expectancy_input)
+    pt_life_expectancy_input = pt_life_expectancy_input.reset_index(drop=True).iloc[:5]
```

Comment:
I think it would be better if the sample data used for testing includes at least one "PT" record to ensure that the filter works correctly.

Reply:
Changed, with using the whole dataframe.